Resources/Meta Analysis/qml/RobustBayesianMetaAnalysis.qml

```diff
@@ -64,38 +64,43 @@ Form
 
 		RadioButton
 		{
-			label: qsTr("Cohen's d / t-statistics & N / SE")
-			value: "cohensd"
-			id: 	measures_cohensd
-			checked:true
+			label:		qsTr("Cohen's d / t-statistics & N / SE")
+			value:		"cohensd"
+			id:			measures_cohensd
+			enabled:	mainWindow.dataAvailable
+			checked:	mainWindow.dataAvailable
```

Comment:
this doesn't work for me, but the same line (`mainWindow.dataAvailable`) does work in the Distributions module to disable sections. @boutinb any ideas?

Reply:
interesting, it works for me on windows. Which part is problematic? The enabling or changing the default selection?